packages/ove-app-audio/src/client/common/audio.js

```diff
@@ -119,7 +118,7 @@ handleStateChange = function (state) {
 
         if (!context.isInitialized) {
             // load a player
-            context.player = new window.OVEHowler();
+            context.player = new window.OVEHowlerPlayer();
```

Comment:
Instead of calling this `OVEHowlerPlayer`, we could call this `OVEHowler` and change the RegEx in the Linter configuration. The reason I used Player in the videos app is because `OVEHTML5Video` or `OVEYouTube` would not have made sense on its own.

Reply:
If OVEYouTube doesn't make sense then neither does OVEHowler. It makes sense to be consistent across apps and use Player since this class does indeed play audio ;)